zubhub_frontend/zubhub/src/views/create_project/script.js

```diff
@@ -693,21 +693,8 @@ export const validationSchema = Yup.object().shape({
     //       ? false
     //       : true;
     //   }),
-    materials_used: Yup.string()
-        .max(10000, 'max')
-        .test('empty', 'required', value => {
-            let is_empty = true;
-
-            value &&
-                value.split(',').forEach(material => {
-                    if (material) {
-                        is_empty = false;
-                    }
-                });
-
-            return !is_empty;
-        }),
-    category: Yup.string(),
+    materials_used: Yup.array(Yup.string()).required(),
```

Comment:
why was this changed to `Yup.array`? the last time I touched this it was `materials_used` was a comma-separated string of items. Does it return real array now? did you check?

Reply:
Earlier, validation wasn't connected directly to the inputs. Of course I checked the returned type then changed it accordingly.

Reply:
I think you should revert all changes like this to their previous version. It's not because the previous version is correct and not buggy, it's because if you try fixing it, then you will be required to fix it accurately and it delays the time it will take to merge your PR. see this for more context https://github.com/unstructuredstudio/zubhub/pull/1089#discussion_r1477551368